app/templates/download_framework_users.html

```diff
@@ -21,11 +21,7 @@
 {% block main_content %}
   <div class="grid-row">
     <div class="column-two-thirds">
-      {%
-        with heading = "Download supplier lists for {}".format(framework['name'])
-      %}
-        {% include "toolkit/page-heading.html" %}
-      {% endwith %}
+      <h1 class="govuk-heading-xl">{{ "Download supplier lists for {}".format(framework['name']) }}</h1>
```

Comment:
We can simplify all these `.format()` calls to:
 `<h1 class="govuk-heading-xl">Download supplier lists for {{ framework['name']) }}</h1>`

Reply:
It's amazing being able to use our templating language as a templating language again.